tfhub_dev/assets/sayakpaul/arbitrary-image-stylization-inceptionv3/fp16/predict/1.md

```diff
@@ -0,0 +1,29 @@
+# Lite sayakpaul/arbitrary-image-stylization-inceptionv3/fp16/predict/1
+TF Lite quantized prediction part of the Arbitrary Style Transfer model.
+
+[![Open Colab Notebok]](https://colab.research.google.com/github/sayakpaul/Adventures-in-TensorFlow-Lite/blob/master/Style_Transfer_Demo_InceptionV3.ipynb)
```

Comment:
The [![Open Colabl Notebook]] line has to between the last defined metadata property (in this case asset-path) and the "### Overview" line.